### ISSUE TITLE
Disable mailchimp on dev an test environments

### ISSUE THREAD

**Issue by smilesrg** [AhmedSamy/HypeMailchimpBundle#31]
Maybe it's a good idea to add option ```disabled_environmetns: [dev, test]``` to return NullMailChimp object when calling hype_mailchimp service on dev and test environments?

**Comment by AhmedSamy** [comment]
definitely, it's a good idea, will implement it once I have time for it, in the meantime, you can always use different config in `config_test` and `config_dev` to different list or different account
